hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/BaseTokenHandler.java

```diff
@@ -427,6 +427,36 @@ public static boolean isMetadataOnlyUpdateOp(@NonNull final TokenUpdateTransacti
         return op.equals(defaultWithMetadata);
     }
 
+    /**
+     * Check if TokenUpdateOp wants to update only some of the low priority keys or the metadata field
+     * low priority keys are -> wipeKey, kycKey, supplyKey, freezeKey, feeScheduleKey, pauseKey or metadataKey
+     */
+    public static boolean noOtherFieldThanLowPriorityKeyOrMetadataWillBeUpdated(
+            @NonNull final TokenUpdateTransactionBody op) {
+        return !((op.symbol() != null && !op.symbol().isEmpty())
+                || (op.name() != null && !op.name().isEmpty())
+                || op.hasTreasury()
+                || op.hasAdminKey()
+                || op.hasAutoRenewAccount()
+                || op.hasAutoRenewPeriod()
+                || op.hasMemo()
+                || op.hasExpiry());
+    }
+
+    /**
+     * Check if a given token already has some of the low priority keys
+     * low priority keys are -> wipeKey, kycKey, supplyKey, freezeKey, feeScheduleKey, pauseKey or metadataKey
+     */
+    public static boolean hasAlreadySomeLowPriorityKeys(@NonNull final Token token) {
```

Comment:
We should consider renaming LowPriorityKeys to NonAdminKeys - as it will be clearer and won't require us to call out explicitly what the other 7 keys are each time.